content/id/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -0,0 +1,109 @@
+                </p>
+
+                <p>Setelah <i>instance</i> aplikasi telah dibuat, Kubernetes Deployment Controller akan melakukan monitoring <i>instance</i> tersebut secara kontinu. Jika <i>instance</i> pada hos Node <i>down</i> atau terhapus, Deployment Controller akan mengganti <i>instance</i> tersebut dengan <i>instance</i> Node lain dalam klaster. <b>Ia menyediakan mekanisme penyembuhan diri (<i>self-healing</i>) untuk mengatasi kegagalan mesin atau pemeliharaan (<i>maintenance</i>)</b></p>
+
+                <p>Pada masa pra-orkestrasi, instalasi <i>script</i> sering digunakan untuk memulai aplikasi, namun cara ini tidak memberikan mekanisme pemulihan ketika terjadi kegagalan mesin. Dengan kemampuan membuat <i>instance</i> aplikasi dan menjaganya tetap berjalan pada Node, Kubernetes Deployment menyediakan manajemen aplikasi dengan pendekatan fundamental yang berbeda. </p>
+
+            </div>
+
+            <div class="col-md-4">
+                <div class="content__box content__box_lined">
+                    <h3>Summary:</h3>
+                    <ul>
+                        <li>Deployment</li>
+                        <li>Kubectl</li>
+                    </ul>
+                </div>
+                <div class="content__box content__box_fill">
+                    <p><i>
+                        Sebuah Deployment bertanggung jawab untuk membuat dan memperbaharui instance dari aplikasi kamu
+                    </i></p>
+                </div>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <h2 style="color: #3771e3;">Men-<i>deploy</i> aplikasi pertama kamu pada Kubernetes</h2>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-md-8">
+                <p><img src="/docs/tutorials/kubernetes-basics/public/images/module_02_first_app.svg"></p>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+
+                <p>Kamu dapat membuat dan mengatur Deployment dengan menggunakan antar muka baris perintah Kubernetes, <b>Kubectl</b>. Kubectl menggunakan Kubernetes API untuk berinteraksi dengan klaster. Pada modul ini, kamu akan belajar perintah-perintah yang sering digunakan Kubectl dalam membuat Deployment untuk menjalankan aplikasi kamu pada klaster Kubernetes.</p>
+
+                <p>Ketika kamu membuat Deployment, kamu perlu mendefinisikan kontainer <i>image</i> untuk aplikasi kamu dan jumlah replika yang kamu inginkan. Kamu dapat mengganti informasi tersebut nanti dengan melakukan pembaharuan Deployment kamu; Modul <a href="docs/tutorials/kubernetes-basics/scale/scale-intro/">5</a> dan <a href="/docs/tutorials/kubernetes-basics/update/update-intro/">6</a> pada <i>bootcamp</i> ini mendiskusikan bagaimana cara melakukan perluasan (<i>scale</i>) dan pembaharuan Deployment kamu.</p>
```

Comment:
Container is API name, no need to translate.
https://kbbi.kemdikbud.go.id/entri/pembaruan

```suggestion
                <p>Ketika kamu membuat Deployment, kamu perlu mendefinisikan Container <i>image</i> untuk aplikasi kamu dan jumlah replika yang kamu inginkan. Kamu dapat mengganti informasi tersebut nanti dengan melakukan pembaharuan Deployment kamu; Modul <a href="docs/tutorials/kubernetes-basics/scale/scale-intro/">5</a> dan <a href="/docs/tutorials/kubernetes-basics/update/update-intro/">6</a> pada <i>bootcamp</i> ini mendiskusikan bagaimana cara melakukan perluasan (<i>scale</i>) dan pembaruan Deployment kamu.</p>
```